server/models/project.js

```diff
@@ -23,6 +23,7 @@ fileSchema.set('toJSON', {
 const projectSchema = new Schema({
   name: { type: String, default: "Hello p5.js, it's the server" },
   user: { type: Schema.Types.ObjectId, ref: 'User' },
+  serveSecue: { type: Boolean, default: false },
```

Comment:
@catarak Should this be enough to have projects saved to the API persist the `serveSecure` flag?

I'm having an issue where saving the project on the client doesn't seem to return it in the response.